commands.go

```diff
@@ -68,13 +68,25 @@ var (
 		{"rm", rmCMD, "???", "???"},
 		{"colour", colorCMD, "???", "This is an alias of color"}, // appease the british
 	}
+	RpcCMDs = []CMD{
+		{"plugins", pluginsCMD, "", "List commands from plugins"},
+	}
+	RpcCMDsRest = []CMD{
+		{"ls-tokens", lsTokensCMD, "", "List all plugin tokens (hashed) and associated data (admin)"},
+		{"revoke-token", revokeTokenCMD, "<token-hash>", "Revoke a plugin token (admin)"},
+		{"grant-token", grantTokenCMD, "<user | data>", "Grant a token and optionally send it to a user (admin)"},
+	}
 )
 
 const (
 	MaxRoomNameLen = 30
 )
 
-func init() {
+func initCMDs() {
+	if Integrations.RPC != nil {
+		MainCMDs = append(MainCMDs, RpcCMDs...)
+		RestCMDs = append(RestCMDs, RpcCMDsRest...)
+	}
```

Comment:
Move these new slice defs into rpc.go and define a new function (`rpcAddCMDS()`) that is used in commands.go's `init()` to add those slices if RPC is enabled.

Reply:
Resolved